lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -299,18 +308,28 @@ def create_or_update_user_script_phases(script_phases, native_target)
           end
 
           def reorder_script_phase(native_target, script_phase, execution_position)
-            return if execution_position == :any
-            compile_build_phase_index = native_target.build_phases.index do |bp|
-              bp.is_a?(Xcodeproj::Project::Object::PBXSourcesBuildPhase)
+            return if execution_position == :any || execution_position.to_s.empty?
+            case execution_position
```

Comment:
can we not do `order_before = case ...`?

Reply:
yep I initially did this to add more functionality but removed that part. will clean this up